### ISSUE TITLE
Cannot install sandbox:2.4-develop or sandbox:master due to composer requirements

### ISSUE THREAD

**Issue by nlzet** [sonata-project/sandbox#476]
_Edited: 2.4-develop seems to be fixed already, see #475  for more information_

Composer install in the master branch now gives me:
```
$ composer install
Loading composer repositories with package information
Installing dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for sonata-project/page-bundle dev-master -> satisfiable by sonata-project/page-bundle[dev-master].
    - sonata-project/page-bundle dev-master requires sonata-project/doctrine-extensions ~1.0 -> no matching package found.
  Problem 2
    - Installation request for sonata-project/notification-bundle dev-master -> satisfiable by sonata-project/notification-bundle[dev-master].
    - sonata-project/notification-bundle dev-master requires sonata-project/doctrine-extensions ~1.0 -> no matching package found.
  Problem 3
    - sonata-project/user-bundle 2.3.x-dev requires sonata-project/doctrine-extensions ~1.0 -> no matching package found.
    - sonata-project/user-bundle 2.2.x-dev requires sonata-project/doctrine-extensions ~1.0 -> no matching package found.
    - sonata-project/user-bundle 2.2.4 requires sonata-project/doctrine-extensions ~1.0 -> no matching package found.
    - sonata-project/user-bundle 2.2.3 requires sonata-project/doctrine-extensions ~1.0 -> no matching package found.
    - sonata-project/user-bundle 2.2.2 requires sonata-project/doctrine-extensions ~1.0 -> no matching package found.
    - sonata-project/user-bundle 2.2.1 requires sonata-project/doctrine-extensions ~1.0 -> no matching package found.
    - sonata-project/user-bundle 2.2.0 requires sonata-project/doctrine-extensions ~1.0 -> no matching package found.
    - sonata-project/news-bundle dev-master requires sonata-project/user-bundle ~2.2 -> satisfiable by sonata-project/user-bundle[2.2.0, 2.2.1, 2.2.2, 2.2.3, 2.2.4, 2.2.x-dev, 2.3.x-dev].
    - Installation request for sonata-project/news-bundle dev-master -> satisfiable by sonata-project/news-bundle[dev-master].

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://groups.google.com/d/topic/composer-dev/_g3ASeIFlrc/discussion> for more details.

Read <http://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

```

**Comment by rande** [comment]
should be fixed. thanks for reporting

**Comment by julienferment** [comment]
Same here.

**Comment by ej-kun** [comment]
They already fixed it. But I started to have a problem with a command that u need to start using the sandbox.
Not sure if it is only for me.
Check this #475 
The problem was fixed in version 2.4

**Comment by ej-kun** [comment]
Also, I think this issue should be closed.
